### ISSUE TITLE
Feature Request(speacil paginators tied to button commands)

### ISSUE THREAD

**Issue by JDJGInc** [oliver-ni/discord-ext-menus-views#6]
![image](https://user-images.githubusercontent.com/22802640/129263263-f08c5a9b-5df3-40c3-88f1-e4fe60c7ca82.png)
![image](https://user-images.githubusercontent.com/22802640/129263386-07ca78bf-db4b-4b6c-9657-1fc477779743.png)
I hope this isn't too hard or annoying but I want a way for me to well, be able to have the ctx thing passed, and maybe a way for it to work inside buttons, I am willing to subclass my interaction or something to get ctx, but I just want a way to add the reactions to a button with content, and also to have an option for empherall messages.
If you don't understand what I mean, basically I just want a way for this library to work inside another button prompt or to somehow get a view response from response.wait(), but the main thing is I want to have an empherall pagintor.
basically this way I can hide stuff, and I never need to DM stuff unless they want some private information that they can see to be Dmed.
